src/fu-remote-list.c

```diff
@@ -36,6 +36,7 @@ struct _FuRemoteList {
 	GPtrArray *array;	  /* (element-type FwupdRemote) */
 	GPtrArray *monitors;	  /* (element-type GFileMonitor) */
 	gboolean testing_remote;
+	gboolean auto_fix;
```

Comment:
Rather than auto fix can you call this "migrate_xz"? 

My logic is that this is the migration now, but we might have more later and need a different option to indicate to do those.

Reply:
Good point, fixed.